apps/emqx_management/test/emqx_mgmt_SUITE.erl

```diff
@@ -397,3 +394,11 @@ mock_print() ->
 
 unmock_print() ->
     meck:unload(emqx_ctl).
+
+safe_unmeck(Module) ->
```

Comment:
For my own curiosity: why was this needed?

Reply:
now that I look at it again. this should work `try meck:unload(module) catch _ : _ -> ok end`
it's a direct exit, but not a 'EXIT' signal.